### PR TITLE
feat: prevent auto merge for manual changes

### PR DESCRIPTION
**Pull request by viestat**
This Pull Request fulfills the following requirements:

- [x] The commit message follows our guidelines.
- [x] Tests for the changes have been added if needed.
- [x] Does not affect documentation or it has been added or updated.

### CODE REVIEW

**Review by merlinnot — COMMENTED** [review]
Should we make it opt-in to avoid breaking changes?

File: README.md [hunk 1]
```diff
@@ -204,6 +204,31 @@ steps:
       MAXIMUM_RETRIES: 2
 ```
 
+### Disable for manual changes
+
+There are cases in which manual changes are needed, for instance, in order to
+make the CI pass or to solve some conflicts that Dependabot (or the bot you are
+using) cannot handle. By default, this Github action will automatically approve
```
Comment (merlinnot) [inline README.md:211]:
```suggestion
using) cannot handle. By default, this GitHub action will automatically approve
```

File: README.md [hunk 2]
```diff
@@ -204,6 +204,31 @@ steps:
...
+and attempt to merge a Pull Request once its triggered regardless of who
+authored the changes on the Pull Request created by
+[`dependabot[bot]`](https://github.com/dependabot) (or the bot you are using).
+This might be undesirable as the author might prefer to get a code review before
+hastily merging the changes.
+
+It is possible to disable this default behavior by setting the value of
+`DISABLED_FOR_MANUAL_CHANGES` to `'TRUE'`.
```
Comment (merlinnot) [inline README.md:219]: We use lowercase in Release Me Action.

```suggestion
`DISABLED_FOR_MANUAL_CHANGES` to `'true'`.
```

File: README.md [hunk 3]
```diff
@@ -204,6 +204,31 @@ steps:
...
+
+```yaml
+steps:
+  - name: Merge me!
+    uses: ridedott/merge-me-action@v1
+    with:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      DISABLED_FOR_MANUAL_CHANGES: 'TRUE'
```
Comment (merlinnot) [inline README.md:227]:
```suggestion
      DISABLED_FOR_MANUAL_CHANGES: 'true'
```

File: README.md [hunk 4]
```diff
@@ -204,6 +204,31 @@ steps:
...
+```
+
+> Important: Please note the single quotes around `TRUE`.
```
Comment (merlinnot) [inline README.md:230]:
```suggestion
> Important: Please note the single quotes around `true`.
```

File: action.yml [hunk 5]
```diff
@@ -22,6 +22,12 @@ inputs:
       'Enable additional functionality to better personalize the behavior. One
       of: DEPENDABOT_MINOR or DEPENDABOT_PATCH.'
     required: false
+  DISABLED_FOR_MANUAL_CHANGES:
+    description:
+      'Disable automatic merges for changes made by a different Github login
+      than the one set in GITHUB_LOGIN. One of: "TRUE" or "FALSE".'
+    default: 'FALSE'
```
Comment (merlinnot) [inline action.yml:29]:
```suggestion
      than the one set in GITHUB_LOGIN. One of: "true" or "false".'
    default: 'false'
```

**Review by merlinnot — COMMENTED** [review]
I'm not 100% sure if that's going to work, I didn't run integration tests myself, but I think since it's opt-in, we can merge it and test on some of our repositories.

File: README.md (hunk 2 above)
Comment (viestat) [reply README.md]: changed in d89acb5

File: README.md (hunk 3 above)
Comment (viestat) [reply README.md]: changed in d89acb5

File: README.md (hunk 4 above)
Comment (viestat) [reply README.md]: changed in d89acb5

File: action.yml (hunk 5 above)
Comment (viestat) [reply action.yml]: changed in d89acb5

**Review by merlinnot — APPROVED** [review]

**Review by joachimroeleveld — APPROVED** [review]

**Pull request merged**